pydoctor/epydoc2stan.py

```diff
@@ -146,14 +146,10 @@ def resolve_identifier_xref(self, identifier: str, lineno: int) -> str:
         if target is not None:
             return target.url
 
-        if identifier == fullID:
-            self.obj.report(
-                "invalid ref to '%s' not resolved" % (identifier,),
-                'resolve_identifier_xref', lineno)
-        else:
-            self.obj.report(
-                "invalid ref to '%s' resolved as '%s'" % (identifier, fullID),
-                'resolve_identifier_xref', lineno)
+        message = f'Cannot find link target for "{fullID}"'
```

Comment:
Maybe also hint for a possible solution

```suggestion
        message = f'Cannot find link target for "{fullID}" (maybe missing intersphinx inventory)'
```

Reply:
That's only one possible reason though. Other reasons could be a typo or pydoctor resolving the identifier incorrectly.

Would this qualify as a FAQ? I've seen the old error message come up in discussions several times, but the confusing phrasing was likely a contributing factor in that.

Another option would be to only display the intersphinx hint if `fullID` has a different root than the software being documented. I'll do a quick experiment to see if this is feasible.

Reply:
> Would this qualify as a FAQ? I've seen the old error message come up in discussions several times, but the confusing phrasing was likely a contributing factor in that.

Yes. I think hat a FAQ can help. It would be the second best... but we can add the link into the error message.

In the idea case, the error message should contain enough information so that you know what step to take to fix it.

Thanks!

Reply:
I ended up implementing the conditional message before merging, see 7095e72.

Reply:
Yep. I saw that. I think that is ok.
But for the future we can keep the error message logic simple and link to a FAQ.

I see this type of error reporting is common with Google / MS / GitHub products.
You can an error message and a web link.